.github/CODEOWNERS

```diff
@@ -145,3 +145,19 @@ sycl/include/sycl/ext/oneapi/bindless* @intel/bindless-images-reviewers
 sycl/source/detail/bindless* @intel/bindless-images-reviewers
 sycl/plugins/unified_runtime/ur/adapters/**/image.* @intel/bindless-images-reviewers
 sycl/test-e2e/bindless_images/ @intel/bindless-images-reviewers
+
+# sycl e2e tests
+sycl/test-e2e/ @intel/llvm-reviewers-runtime
+sycl/test-e2e/GroupAlgorithm/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/GroupLocalMemory/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/SubGroup/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/SubGroupMask/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/Plugin/*level-zero* @intel/dpcpp-l0-pi-reviewers
+sycl/test-e2e/ESIMD/ @intel/dpcpp-esimd-reviewers
+sycl/test-e2e/InvokeSimd/ @intel/dpcpp-esimd-reviewers
+sycl/test-e2e/BFloat16/ @intel/dpcpp-tools-reviewers
+sycl/test-e2e/AOT/ @intel/dpcpp-tools-reviewers
+sycl/test-e2e/DeviceCodeSplit/ @intel/dpcpp-tools-reviewers
+sycl/test-e2e/SeparateCompile/ @intel/dpcpp-tools-reviewers
+sycl/test-e2e/Printf/ @intel/dpcpp-tools-reviewers
```

Comment:
dpcpp-tools-reviewers does not seem to be the right match here. Thanks

Reply:
Per discussion above, we are going to go with two owners for some of these, and see how that works out.